### ISSUE TITLE
[ACT] Bug - fix rule "WCAG21_Label_Accessible" to not report fail when no visible label text required

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1157]
Engine rule [WCAG21_Label_Accessible.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG21_Label_Accessible.ts)

Rule correctly reports against requirement [2.5.3: Label in Name (Level A)](https://www.ibm.com/able/requirements/requirements/#2_5_3) - For user interface components with labels that include text or images of text, the accessible name contains the text that is presented visually

**_Why?_** improve consistency with proposed rule ACT 2ee8b8 [Visible label is part of accessible name](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/2ee8b8/proposed/) and fix false negative.

- [x] Fix rule to not report a failure for [ACT 2ee8b8-Passed Example 6](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/2ee8b8/efa9543339cdad5412c7719b266a633a29ce149e.html)
- [x] Ensure rule maps to [ACT 2ee8b8](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/2ee8b8/proposed/)
- [x] No change needed to Message & Help: Violation — [Accessible name does not match or contain the visible label text](https://unpkg.com/accessibility-checker-engine@3.1.40/help/en-US/WCAG21_Label_Accessible.html#%7B%22message%22%3A%22Accessible%20name%20does%20not%20match%20or%20contain%20the%20visible%20label%20text%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22VIOLATION%22%2C%22FAIL%22%5D%2C%22reasonId%22%3A%22Fail_1%22%7D)

### Background
- See rule discussion on box [ACT 2ee8b8–Visible label is part of accessible name](https://ibm.ent.box.com/file/1062102497451)
- See **_Details by test case_** tab in the spreadsheet [ACT vs Engine inconsistencies.xlsx](https://ibm.ent.box.com/file/1040535127435?s=9o6m83gzzc9sp8pu8lwtyesh0b8wwxln) on box.

**Comment by philljenkins** [comment]
@shunguoy this issue is a dup of #1158. I believe this issue was created in a new-ish template
Closing as a dup
